assets/css/styles.css

```diff
@@ -0,0 +1,311 @@
+
+.completed-projects-section-content, 
+.experience-section-content {
+  flex: 5;
+}
+.completed-projects-section-content > ul, 
+.experience-section-content > ul {
+  display: flex;
+  flex-wrap: wrap;
+  list-style-type: none;
+  padding: 5px;
+}
+
+.completed-projects-section-content > ul > li, 
+.experience-section-content > ul > li {
+  height: 20vh;
+  flex-grow: 1;
+  border: solid 1px #9ef01a;
+}
+
+.experience-section-content > ul > li{
+  height: 3vh;
+}
+
+.experience-section-content-text{
+  margin: 2px;
+}
+
+.completed-projects-section-content-img {
+  max-height: 100%;
+  min-width: 100%;
+  object-fit: cover;
+  vertical-align: bottom;
+  object-fit:fill;
+  
+}
+
+.completed-projects-section-content > ul > li:first-child{
+  width: 100%;
+  height: 25vh; 
+}
+
+.completed-projects-section-content > ul > li:last-child, 
+.experience-section-content > ul > li:last-child {
+  flex-grow: 10;
+  text-align: center;
+}
+
+.completed-projects-section-content-img:hover {
+  position:relative;
+  top:-25px;
+  left:-35px;
+  width:500px;
+  height:auto;
+  display:block;
+  z-index:999;
+}
+
+/* code that affects the logo ims for the contact section*/
+
+.contact-section-content-logo {
+  flex-grow: 1;
+  text-align: center;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.hoverwrap {
+  position: relative; 
+  max-width: 700px; 
+}
+
+.hoverwrap > img { width: 100%; }
+
+
+.hovercap {
+  box-sizing: border-box;
+  width: 100%;
+  padding: 10px;
+  position: absolute;
+  bottom: 0; left: 0;
+  background-color: rgba(1, 19, 5, 0.712);
+  color: #70e000;
+}
+
+.hovercap {
+  visibility: none; opacity: 0;
+  transition: opacity 0.3s;
+}
+
+.hoverwrap:hover .hovercap {
+  visibility: visible; opacity: 1;
+}
+
+
+/* code that affects the main content footer section*/
+
+footer {
+  color: #006400;
+  text-align: center;
+}
+
+
+
+/* mobile viewport styles */
+@media screen and (max-width: 900px) {
+  /* responsive code for the nav bar*/
+  .navigation-bar {
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .navigation-bar-profile {
+    border-bottom: solid 3px #38b000;
+    border-right: 0px;
+  }
+  .navigation-bar-links {
+    flex-direction: column;
+    align-items: center;
+    margin: 10px;
+  }
+
+  .nav-link-item > a {
+    padding: 2px;
+  }
+
+  .profile-img {
+    width: 90px;
+    height: 90px;
+  }
+
+  /* responsive code for each main content section*/
+
+  .about-section, 
+  .experience-section,
+  .completed-projects-section,
+  .contact-section {
+    display: flex;
+    flex-direction: column;
+  }
+
+  /* responsive code for each main content title */
+  .about-section-title > h1, 
+  .experience-section-title > h2,
+  .completed-projects-section-title > h3,
+  .contact-section-title > h4 {
+    text-align: center;
+  }
+
+
+  .completed-projects-section-content > ul {
+    padding: 5px;
+  }
+
+  .completed-projects-section-content > ul > li {
+    height: 35vh;
```

Comment:
Consider changing the width here also. This may be why the site isn't completely responsive.